service/src/test/java/uk/gov/hmcts/reform/fpl/service/ManageHearingsServiceTest.java

```diff
@@ -664,19 +678,6 @@ void shouldSendNoticeOfHearingIfRequested() {
             NOTICE_OF_HEARING.getDocumentTitle(time.now().toLocalDate()));
     }
 
-    @Test
-    void shouldNotSendNoticeOfHearingIfNotRequested() {
```

Comment:
is NOH always sent now or why is this one removed?

Reply:
After some checking with UX, I added the page asking if the NOH should be sent back in so this test is no longer needed